build.sbt

```diff
@@ -5490,12 +5490,17 @@ lazy val runProjectManagerDistribution =
     "Run or --debug the project manager distribution with arguments"
   )
 runProjectManagerDistribution := {
-  buildEngineDistribution.value
-  buildProjectManagerDistribution.value
+  buildEngineDistributionNoIndex.value
+  if (GraalVM.EnsoLauncher.native) {
+    // how do I do the following only when `ENSO_LAUNCHER=native`?
+    // buildProjectManagerDistributionCond.value
```

Comment:
@Akirathan, as our `sbt` expert, can you please advice how to do the `buildProjectManagerDistribution` dependency conditionally? Just uncommenting and using `buildProjectManagerDistribution.value` doesn't work. Thank you for your advice.

Reply:
I have pushed 0402f9325e94a6a5ae52f3bf71e11f2e820403a2 . With that it works just as you would like it to. I have tried it locally - by default, `runProjectManagerDistribution` does not trigger native image build, unless you have `ENSO_LAUNCHER=native` in env var.

Reply:
Looks simple when one knows to use `Def.taskIf`! Thanks a lot. Now you need to approve your change.